nilearn/signal.py

```diff
@@ -510,3 +515,84 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         signals *= np.sqrt(signals.shape[0])  # for unit variance
 
     return signals
+
+
+def clean_img(niimg, sessions=None, detrend=True, standardize=True,
+          confounds=None, low_pass=None, high_pass=None, t_r=2.5):
+    """Improve SNR on masked fMRI signals.
+
+       This function can do several things on the input signals, in
+       the following order:
+
+       - detrend
+       - standardize
+       - remove confounds
+       - low- and high-pass filter
+
+       Low-pass filtering improves specificity.
+
+       High-pass filtering should be kept small, to keep some
+       sensitivity.
+
+       Filtering is only meaningful on evenly-sampled signals.
+
+       Parameters
+       ==========
+       imgs: Niimg-like object
+            See http://nilearn.github.io/manipulating_images/manipulating_images.html#niimg.
+            4D image. The signals in the last dimension are filtered.
+
+       sessions : numpy array, optional
+           Add a session level to the cleaning process. Each session will be
+           cleaned independently. Must be a 1D array of n_samples elements.
+
+       confounds: numpy.ndarray, str or list of
+           Confounds timeseries. Shape must be
+           (instant number, confound number), or just (instant number,)
+           The number of time instants in signals and confounds must be
+           identical (i.e. signals.shape[0] == confounds.shape[0]).
+           If a string is provided, it is assumed to be the name of a csv file
+           containing signals as columns, with an optional one-line header.
+           If a list is provided, all confounds are removed from the input
+           signal, as if all were in the same array.
+
+       t_r: float
+           Repetition time, in second (sampling period).
+
+       low_pass, high_pass: float
+           Respectively low and high cutoff frequencies, in Hertz.
+
+       detrend: bool
+           If detrending should be applied on timeseries (before
+           confound removal)
+
+       standardize: bool
```

Comment:
Minor detail: it would be nice if doc too follows the same order as described above.